src/wireviz/wireviz.py

```diff
@@ -311,28 +312,28 @@ def main():
 
     args = parse_cmdline()
 
-    if not os.path.exists(args.input_file):
-        print(f'Error: input file {args.input_file} inaccessible or does not exist, check path')
+    file_in = Path(args.input_file)
+
+    if not file_in.exists():
+        print(f'Error: input file {file_in} inaccessible or does not exist, check path')
         sys.exit(1)
 
-    with open_file_read(args.input_file) as fh:
+    with open_file_read(file_in) as fh:
         yaml_input = fh.read()
 
     if args.prepend_file:
-        if not os.path.exists(args.prepend_file):
+        if not Path(args.prepend_file).exists():
```

Comment:
For some strange reason, this suggestion was not included in my review just now:
(The same argument as @laurierloi describes above also applies here.)
```suggestion
        if not Path(args.prepend_file).is_file():
```